### PR TITLE
827159 UG documentation for the step area series of .net MAUI

### PR DESCRIPTION
**Pull request by EzhilarasanElangovan31**
### Feature description ###

Added User guide for StepArea chart in MAUI-Cartesian chart.

### Output ###
maui_stepArea_chart

### CODE REVIEW

File: MAUI/Cartesian-Charts/StepArea.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+chart.Series.Add(series);
+this.Content= chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Marker customization
+
+To change the series markers appearance, create an instance of the [MarkerSettings](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.AreaSeries.html#Syncfusion_Maui_Charts_AreaSeries_MarkerSettings) property. The following properties are used to customize marker appearance.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/StepArea.md:103]: Link was wrong, remove link.

File: MAUI/Cartesian-Charts/StepArea.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+chart.Series.Add(series);
+this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/StepArea.md:164]: Remove unwanted lines.

File: MAUI/Cartesian-Charts/StepArea.md (hunk 1 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/StepArea.md:103]: Link removed

File: MAUI/Cartesian-Charts/StepArea.md (hunk 2 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/StepArea.md]: Removed unwanted lines

**Review by Saravanan-Madhesh — APPROVED** [review]
The changes are fine.

**Pull request merged**